### PR TITLE
[TW-234] Doc enterprise app for macOS 

### PR DESCRIPTION
**Pull request by jkonrath-postman**
(Needs screenshot at top)

### CODE REVIEW

File: src/pages/docs/administration/managing-enterprise-deployment.md [hunk 1]
```diff
@@ -48,7 +55,7 @@ To request this change, reach out to your Postman Account Manager or [contact Po
 
 ## Deploying the Postman Enterprise app
 
-Postman's Enterprise Application is a variant of Postman’s Desktop Application that offers greater control to administrators looking to deploy Postman at an enterprise level. It is available as an MSI package for Windows and supports silent installation, system-wide installation, and additional configurations to control how Postman is installed on users' devices.
+Postman's Enterprise Application is a variant of Postman’s Desktop Application that offers greater control to administrators looking to deploy Postman at an enterprise level. It is available as an MSI package for Windows and PKG package for macOS, and supports silent installation, system-wide installation, and additional configurations to control how Postman is installed on users' devices.
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/managing-enterprise-deployment.md:58]: Change "Application" to lower case? Elsewhere on the page "app" is lower case

File: src/pages/docs/administration/managing-enterprise-deployment.md [hunk 2]
```diff
@@ -48,7 +55,7 @@ To request this change, reach out to your Postman Account Manager or [contact Po
 
 ## Deploying the Postman Enterprise app
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/managing-enterprise-deployment.md:56]: I think nesting all subsequent headings under "Deploying the Postman Enterprise app" makes the page harder to scan, especially with the way the H4's render. I would move all headings after this one up a level

File: src/pages/docs/administration/managing-enterprise-deployment.md [hunk 3]
```diff
@@ -106,30 +113,120 @@ It is always recommended to [run silent installations with logging enabled](#ena
...
+PKG installer packages have the following capabilities:
+
+* You can select between system-wide, per-user installation, or custom installation disks
+* They do not require reboots after installation
+* They are fully configurable for silent installation
+* Apple Silicon is supported
+
+#### Installing a PKG installer package
+
+A PKG installer can be installed by double-clicking on the file and following the interactive configurable wizard. PKGs can also be installed from a command-line interface using the macOS `installer(8)` tool.
```
Comment (mark-anthony-postman) [inline src/pages/docs/administration/managing-enterprise-deployment.md:157]: Are the numbers in parentheses needed for `installer(8)`, `defaults(1)`, and `pkgutil(1)`? Doing some searches, I see those on some `man` pages but I don't think they're part of the actual command

File: src/pages/docs/administration/managing-enterprise-deployment.md (hunk 1 above)
Comment (jkonrath-postman) [reply src/pages/docs/administration/managing-enterprise-deployment.md:58]: It should be app, but if you take a look at the screenshot, both are used on the download page. I'm sticking to app to be consistent.

File: src/pages/docs/administration/managing-enterprise-deployment.md (hunk 3 above)
Comment (jkonrath-postman) [reply src/pages/docs/administration/managing-enterprise-deployment.md:157]: None of these are commands with man pages in multiple sections, so I'm striking those from each occurrence.

File: src/pages/docs/administration/managing-enterprise-deployment.md [hunk 4]
```diff
@@ -74,62 +81,152 @@ For example, you can run the following command to perform a system-wide installa
...
+## Installing the Postman Enterprise macOS app
+
+macOS supports a flexible installer technology called PKG that can be easily installed without user intervention in scripting or enterprise deployment scenarios.
+
+PKG installer packages have the following capabilities:
+
+* You can select between system-wide, per-user installation, or custom installation disks
+* They do not require reboots after installation
+* They are fully configurable for silent installation
+* Apple Silicon is supported
```
Comment (HeatherStenson) [inline src/pages/docs/administration/managing-enterprise-deployment.md:153]: It looks like these should end in periods, per the LC style guide ("If a bulleted list has a mix of complete and incomplete sentences, all sentences end in a period."):

```suggestion
* You can select between system-wide, per-user installation, or custom installation disks.
* They do not require reboots after installation.
* They are fully configurable for silent installation.
* Apple Silicon is supported.
```

File: src/pages/docs/administration/managing-enterprise-deployment.md [hunk 5]
```diff
@@ -74,62 +81,152 @@ For example, you can run the following command to perform a system-wide installa
...
+
+A PKG installer can be installed by double-clicking on the file and following the interactive configurable wizard. PKGs can also be installed from a command-line interface using the macOS `installer` tool.
+
+The `LocalSystem` target instructs the PKG to install app bundles to `/Applications` and store installation settings at `/var/root/Library/Preferences`:
+
+``` shell
+sudo installer -dumplog -verbose -pkg path/to/app.pkg -target LocalSystem
+```
+
+You can also perform a per-user installation by specifying the `CurrentUserHomeDirectory` target. Given this target, the PKG installs app bundles to `$HOME/Applications` and store installation settings at `$HOME/Library/Preferences`:
```
Comment (HeatherStenson) [inline src/pages/docs/administration/managing-enterprise-deployment.md:165]:
```suggestion
You can also perform a per-user installation by specifying the `CurrentUserHomeDirectory` target. Given this target, the PKG installs app bundles to `$HOME/Applications` and stores installation settings at `$HOME/Library/Preferences`:
```

**Review by HeatherStenson — APPROVED** [review]
Looks good! 🍊

**Review by RaisinTen — CHANGES_REQUESTED** [review]

File: src/pages/docs/administration/managing-enterprise-deployment.md [hunk 6]
```diff
@@ -74,62 +81,152 @@ For example, you can run the following command to perform a system-wide installa
...
+
+``` shell
+installer -dumplog -verbose -pkg path/to/app.pkg -target CurrentUserHomeDirectory
+```
+
+### PKG installation options
+
+You can define custom properties with the `installationOptions` profile setting. These settings can be changed at installation time using the macOS `defaults` tool either before or after installing the PKG.
```
[the remark this answers is not in the document]
Comment (jkonrath-postman) [reply src/pages/docs/administration/managing-enterprise-deployment.md:173]: OK, I'm removing this section then.

[the remark this answers is not in the document]
Comment (RaisinTen) [reply src/pages/docs/administration/managing-enterprise-deployment.md]: @jkonrath-postman the rest of the section is still useful. The PKG users are still supposed to use `defaults` to set the custom values in the system-wide/user-specific plist. Could you please add back that section and apply the suggested change?

[the remark this answers is not in the document]
Comment (jkonrath-postman) [reply src/pages/docs/administration/managing-enterprise-deployment.md]: @RaisinTen I've added back the section, and just removed the first sentence. Is that correct?

**Pull request merged**